### PR TITLE
Consolidate node configurations

### LINKED ISSUE #35: Consolidate node configurations
At first glance, at least two source files specifies default node:
* `utils/cert.js:var` defaultNode = "http://node.deviceproof.org:14265";
* `utils/mamUtils.js:var` iota = new IOTA({ provider: 'http://node.deviceproof.org:14265' })

We shall avoid hardcoded initialization of each IOTA object. Instead, system-wide configuration file can be used accordingly.

### PR DESCRIPTION
**Pull request by ender503**
Setup the IOTA object by using system-wide configuration, and move the
setup process into 'iotaSetup.js'. Close #35

### CODE REVIEW

File: utils/cert.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
-var IOTA = require('iota.lib.js')
 var crypto = require('crypto')
 
-var defaultNode = "http://node.deviceproof.org:14265";
-
-var iota = new IOTA({
-  provider: defaultNode
-});
+const iota = require('./iotaSetup.js');
```
Comment (jserv) [inline utils/cert.js]: You shall mention how to generate `iotaSetup.js` along with top-level file `README.md`.

File: README.md [hunk 2]
```diff
@@ -51,15 +51,26 @@ device is lost.
...
 ## Get started
 
-- [x] Create new file named `.env` and specify `BACKEND` and `API_HOST`
-  * `BACKEND`: an address (IP or domain) providing a proxy to bypass CORS.
+### Setup environment
+
+TangleID use [swarm node](https://github.com/yillkid/iota-swarm-node) as backend to interact with Tangle.
+
+- Create new file named `.env` and specify `BACKEND` and `API_HOST`
+  * `BACKEND`: an address (IP or domain) to the smarm node.
```
Comment (jserv) [inline README.md]: Typo. Should be `swarm`.

File: README.md [hunk 3]
```diff
@@ -51,15 +51,26 @@ device is lost.
 
 ## Get started
 
-- [x] Create new file named `.env` and specify `BACKEND` and `API_HOST`
-  * `BACKEND`: an address (IP or domain) providing a proxy to bypass CORS.
+### Setup environment
+
+TangleID use [swarm node](https://github.com/yillkid/iota-swarm-node) as backend to interact with Tangle.
```
Comment (jserv) [inline README.md]: Syntax error: `uses`.

**Comment by ender503** [comment]
Typo fixed. 👍

**Comment by jserv** [comment]
Can you provide failback API host(s) if environment variables are not valid?

**Comment by ender503** [comment]
Maybe we could assign the default value to the environment variables? 🤔

**Comment by jserv** [comment]
Regardless of the methods to preserving default values, it is important not to depend on poor documentations for assigning them. That is, default values should be explicitly passed via certain approaches such as environment values.

**Comment by jserv** [comment]
Any failback implementation available?

**Comment by ender503** [comment]
I try to preserve the default environment values in the Next.js for now.

It is weird to load the environment values in the Next.js configuration,
we should load the environment in the 'server.js' when we remove the Next.js.

**Pull request merged**